test/decl/protocol/conforms/nscoding.swift

```diff
@@ -0,0 +1,74 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -parse-as-library -swift-version 4 %s -verify
+
+// REQUIRES: objc_interop
+
+import Foundation
+
+// Top-level classes
+class CodingA : NSObject, NSCoding {
```

Comment:
Please add a test case for a subclass of a class that already conforms to NSCoding.